### PR TITLE
https://issues.redhat.com/browse/ACM-10437

### PR DESCRIPTION
**Pull request by oafischer**
2.10 only

### CODE REVIEW

File: networking/submariner/deploy_subm_console.adoc [hunk 1]
```diff
@@ -6,11 +6,16 @@ Before you deploy Submariner with {product-title}, you must prepare the clusters
 * Amazon Web Services
 * Google Cloud Platform
 * IBM Power Systems Virtual Server
+* Red Hat OpenShift Kubernetes Service
```
Comment (yboaron) [inline networking/submariner/deploy_subm_console.adoc]: I think ROKS name changed to Red Hat OpenShift on IBM Cloud (RHOIC),
https://www.ibm.com/products/openshift

Comment (oafischer) [reply networking/submariner/deploy_subm_console.adoc]: You are right, I'll update it

File: networking/submariner/deploy_subm_console.adoc [hunk 2]
```diff
@@ -6,11 +6,16 @@ Before you deploy Submariner with {product-title}, you must prepare the clusters
 * Amazon Web Services
 * Google Cloud Platform
 * IBM Power Systems Virtual Server
+* Red Hat OpenShift on IBM Cloud
```
Comment (yboaron) [inline networking/submariner/deploy_subm_console.adoc]: Where do we specify that status is Tech Preview and not GAed ?

Comment (oafischer) [reply networking/submariner/deploy_subm_console.adoc]: Updated

**Review by yboaron — APPROVED** [review]

**Comment by yboaron** [comment]
cc @mihivagyok

**Review by jc-berger — APPROVED** [review]

**Pull request merged**